drivers/nrf9160_gps/nrf9160_gps.c

```diff
@@ -428,20 +428,37 @@ static int init(struct device *dev)
 	init_thread(dev);
 
 	#if CONFIG_NRF9160_GPS_SET_MAGPIO
-		int err;
-		char buf[50] = {0};
-
-		err = at_cmd_write(CONFIG_NRF9160_GPS_MAGPIO_STRING,
-				buf, sizeof(buf), NULL);
-		if (err) {
-			LOG_ERR("Could not confiugure MAGPIO, error: %d", err);
-			return err;
+		int magpio_err;
```

Comment:
I suggest simply `err` and `buf`, here and below.

Reply:
Done, changed to `err`